### ISSUE TITLE
Git quickstart

### ISSUE THREAD

**Issue by pvalena** [developer-portal/content#377]
I've noticed we're missing a git quickstart.

I've drafted some in:

https://github.com/developer-portal/website/blob/master/DEVELOPMENT.md

(Where we probably need one, or link to a one.)

**Comment by pvalena** [comment]
@hhorak WDYT?

**Comment by hhorak** [comment]
It might be useful for less technical people, so why not. I think the user in the clone command might be also replaced (currently `pvalena` is mentioned there): https://github.com/developer-portal/website/blame/master/DEVELOPMENT.md#L87
Otherwise LGTM

**Comment by pvalena** [comment]
> It might be useful for less technical people, so why not. I think the user in the clone command might be also replaced (currently `pvalena` is mentioned there): https://github.com/developer-portal/website/blame/master/DEVELOPMENT.md#L87
> Otherwise LGTM

Good catch- that's a mistake.

Thanks for checking

**Comment by pvalena** [comment]
I'll take a look at it then, which section do you think it should go to? `Get Tools` or `Deploy and Distribute` ? :)

**Comment by pvalena** [comment]
@jackorp WDYT?

**Comment by jackorp** [comment]
I'd say it belongs to the `Get Tools` section as Git is a versioning tool and it would IMHO look out of place in between Openshift RPM and Flatpak and whatnot.

**Comment by pvalena** [comment]
> I'd say it belongs to the `Get Tools` section as Git is a versioning tool and it would IMHO look out of place in between Openshift RPM and Flatpak and whatnot.

You're right, it's not much of a deploy/distribution tool, outside of source code :). `Tools it is`. And the guide itself? Would you add something? Can you try it?

**Comment by pvalena** [comment]
The `quay.io` container builds are working! https://quay.io/repository/developer-portal/devel?tab=builds

**Comment by jackorp** [comment]
> > I'd say it belongs to the `Get Tools` section as Git is a versioning tool and it would IMHO look out of place in between Openshift RPM and Flatpak and whatnot.
> 
> You're right, it's not much of a deploy/distribution tool, outside of source code :). `Tools it is`. And the guide itself? Would you add something? Can you try it?

I'll try it later this weekend.

However, from a glance, I don't see setting a name or email? If someone is new to git they probably don't know this needs to be done.

I am also missing adding and committing changes.

**Comment by pvalena** [comment]
You're right, ... I'd assume people know that, but then again, it would be a bad guide :). I'll add something like that, and create a PR. Thanks!
